app/assets/src/components/Samples.jsx

```diff
@@ -2,8 +2,12 @@ import React from "react";
 import axios from "axios";
 import ReactDOM from "react-dom";
 import moment from "moment";
+import cx from "classnames";
 import $ from "jquery";
 import Materialize from "materialize-css";
+import { intersection, union, difference, map, keyBy, take } from "lodash";
```

Comment:
Do we really need to use lodash? I am asking because I have been pushing for not using too many extra libraries. I believe we are more bloated then we should.
I guess intersection might be hard to replace... but you probably do not need the merge below.

What do you mean with the comment below?

Reply:
In my experience, lodash has been very convenient and allows developers to write more concise code (though there is a bit of a learning curve).

I would prefer to keep it. There are ways to reduce the bundle size (at the expense of slightly longer build times): https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark

Is bundle size a serious issue? lodash is one of the few libraries I routinely use.

Reply:
I believe bundle size could become a factor in some sites and if we do not carefully evaluate before adding packages.
On the other hand, `lodash` does seem useful and does allow for custom builds if we ever need to optimize so I think it is fine.

I guess my next question why do we need `lodash/fp`?